langgraph/src/checkpoint/sqlite.ts

```diff
@@ -122,31 +122,31 @@ CREATE TABLE IF NOT EXISTS checkpoints (
 
   async *list(config: RunnableConfig): AsyncGenerator<CheckpointTuple> {
     this.setup();
-    const threadId = config.configurable?.threadId;
+    const thread_id = config.configurable?.thread_id;
 
     try {
       const rows: Row[] = this.db
         .prepare(
-          `SELECT thread_id, thread_ts, parent_ts, checkpoint, metadata FROM checkpoints WHERE thread_id = ? ORDER BY thread_ts DESC`
+          `SELECT thread_id, checkpoint_id, parent_id, checkpoint, metadata FROM checkpoints WHERE thread_id = ? ORDER BY checkpoint_id DESC`
         )
-        .all(threadId) as Row[];
+        .all(thread_id) as Row[];
 
       if (rows) {
         for (const row of rows) {
           yield {
             config: {
               configurable: {
-                threadId: row.thread_id,
-                threadTs: row.thread_ts,
+                thread_id: row.thread_id,
+                checkpoint_id: row.checkpoint_id,
               },
             },
             checkpoint: this.serde.parse(row.checkpoint) as Checkpoint,
             metadata: this.serde.parse(row.metadata) as CheckpointMetadata,
-            parentConfig: row.parent_ts
+            parentConfig: row.parent_id
               ? {
                   configurable: {
-                    threadId: row.thread_id,
-                    threadTs: row.parent_ts,
+                    thread_id: row.thread_id,
```

Comment:
so we're officially dropping support for pascal case then?

Reply:
In things that are meant to be sent over the wire / could one day be shared between js and py yes